configurations/kalobeyei_pipeline_config.json

```diff
@@ -122,25 +119,64 @@
-    {"RapidProKey": "Child_Gender (Text) - wusc_leap_s01_kalobeyei_demogs", "PipelineKey": "child_gender_raw"},
-    {"RapidProKey": "Child_Gender (Time) - wusc_leap_s01_kalobeyei_demogs", "PipelineKey": "child_gender_time"},
-    {"RapidProKey": "Consent_To_Engage_Child (Text) - wusc_leap_s01_kalobeyei_demogs", "PipelineKey": "consent_to_engage_child_raw"},
-    {"RapidProKey": "Consent_To_Engage_Child (Time) - wusc_leap_s01_kalobeyei_demogs", "PipelineKey": "consent_to_engage_child_time"},
-    {"RapidProKey": "Age_Of_Parent (Text) - wusc_leap_s01_kalobeyei_demogs", "PipelineKey": "age_of_parent_raw"},
-    {"RapidProKey": "Age_Of_Parent (Time) - wusc_leap_s01_kalobeyei_demogs", "PipelineKey": "age_of_parent_time"},
-    {"RapidProKey": "Currently_In_School (Text) - wusc_leap_s01_kalobeyei_demogs", "PipelineKey": "currently_in_school_raw"},
-    {"RapidProKey": "Currently_In_School (Time) - wusc_leap_s01_kalobeyei_demogs", "PipelineKey": "currently_in_school_time"}
+    {"RapidProKey": "Participants_Engaging (Text) - wusc_leap_s01_kalobeyei_demogs", "PipelineKey": "kalobeyei_participants_engaging_raw"},
+    {"RapidProKey": "Participants_Engaging (Time) - wusc_leap_s01_kalobeyei_demogs", "PipelineKey": "kalobeyei_participants_engaging_time"},
+    {"RapidProKey": "Targeted_Group_Parent (Text) - wusc_leap_s01_kalobeyei_demogs", "PipelineKey": "kalobeyei_targeted_group_parent_raw"},
+    {"RapidProKey": "Targeted_Group_Parent (Time) - wusc_leap_s01_kalobeyei_demogs", "PipelineKey": "kalobeyei_targeted_group_parent_time"},
+    {"RapidProKey": "Child_Gender (Text) - wusc_leap_s01_kalobeyei_demogs", "PipelineKey": "kalobeyei_child_gender_raw"},
+    {"RapidProKey": "Child_Gender (Time) - wusc_leap_s01_kalobeyei_demogs", "PipelineKey": "kalobeyei_child_gender_time"},
+    {"RapidProKey": "Consent_To_Engage_Child (Text) - wusc_leap_s01_kalobeyei_demogs", "PipelineKey": "kalobeyei_consent_to_engage_child_raw"},
+    {"RapidProKey": "Consent_To_Engage_Child (Time) - wusc_leap_s01_kalobeyei_demogs", "PipelineKey": "kalobeyei_consent_to_engage_child_time"},
+    {"RapidProKey": "Age_Of_Parent (Text) - wusc_leap_s01_kalobeyei_demogs", "PipelineKey": "kalobeyei_age_of_parent_raw"},
+    {"RapidProKey": "Age_Of_Parent (Time) - wusc_leap_s01_kalobeyei_demogs", "PipelineKey": "kalobeyei_age_of_parent_time"},
+    {"RapidProKey": "Currently_In_School (Text) - wusc_leap_s01_kalobeyei_demogs", "PipelineKey": "kalobeyei_currently_in_school_raw"},
+    {"RapidProKey": "Currently_In_School (Time) - wusc_leap_s01_kalobeyei_demogs", "PipelineKey": "kalobeyei_currently_in_school_time"}
   ],
 
   "PipelineName": "leap_kalobeyei_s01_pipeline",
   "ProjectStartDate": "2020-01-18T13:00:00+03:00",
   "ProjectEndDate": "2100-01-01T00:00:00+03:00",
   "FilterTestMessages": true,
   "MoveWSMessages": true,
+  "AutomatedAnalysis": {
+    "TrafficLabels": [
+      {"StartDate": "2021-02-19T19:30+03:00", "EndDate": "2020-02-20T16:30+03:00", "Label": "E01 Ad"},
```

Comment:
Does RDA agree with this definition? It's quite a long time for an advert

Reply:
Yes, this is okay for the camp context and based on past experiences. Promos have a negligible impact on engagement but we have people responding to the advert even after 12 hour period.